roles/principal_data_engineer.md

```diff
@@ -0,0 +1,59 @@
+# Principal Data Engineer
+
+UK-based in Birmingham, Bristol, Glasgow, London, Manchester and Newcastle.
+
+Please apply for this role at [www.madetech.com/careers](http://www.madetech.com/careers).
+
+Our Principal Data Engineers enable public sector organisations to transform service delivery with data to improve society, for everyone. They provide our most senior data leadership and support to our clients and teams, build and manage key relationships with clients, manage data consultancy and delivery across one or more accounts, and support sales.
+
+## What does the job entail?
+
+At Made Tech we want to positively impact the future of the country by using technology to improve society, for everyone. We want to empower the public sector to deliver and continuously improve digital services that are user-centric, data-driven and freed from legacy technology. To do this we work alongside brilliant public servants to drive better decisions through data and automation.
+
+As a Principal Data Engineer you will work closely with our customers senior leadership to help inform their data strategy and to ensure our teams are delivering to a high quality and are aligned to our customers data vision. Working alongside delivery, client and other capability principals, you will help shape account plans and be responsible for delivering against them.
```

Comment:
Minor structural stuff, plus I believe the current stance is to prefer "client" over "customer":

```suggestion
As a Principal Data Engineer you will work closely with our clients' senior leadership teams to help inform their data strategy, as well as ensuring our teams are delivering to a high quality and aligned to our clients' data vision. Working alongside delivery, client and other capability principals, you will help shape account plans and be responsible for delivering against them.
```